inst/qml/SEM.qml

```diff
@@ -143,9 +99,9 @@ Form
 				]
 			}
 			CheckBox { name: "meanStructure";					label: qsTr("Include mean structure") ; checked: eq_intercepts.checked || eq_means.checked || eq_thresholds.checked
-				CheckBox { name: "latentInterceptFixedToZero";		label: qsTr("Fix latent means to zero"); checked: !eq_means.checked }
-				CheckBox { name: "manifestInterceptFixedToZero";	label: qsTr("Fix manifest intercepts to zero"); }
-				CheckBox { name: "manifestMeanFixedToZero";		label: qsTr("Fix mean of manifest intercepts to zero")}
+			CheckBox { name: "latentInterceptFixedToZero";		label: qsTr("Fix latent means to zero"); checked: !eq_means.checked }
+			CheckBox { name: "manifestInterceptFixedToZero";	label: qsTr("Fix manifest intercepts to zero"); }
```

Comment:
```suggestion
			CheckBox { name: "manifestInterceptFixedToZero";	label: qsTr("Fix manifest intercepts to zero")}
```

Reply:
😅

Reply:
also, I just noticed that the option name does not correspond to the label,
the correct version is fixManifestInterceptToZero

Reply:
true, but the rules when we did the option renaming was not verbs in front.

Reply:
"Option names should be nouns, because they are things, whereas functions names should start with a verb, because they do things."

Reply:
okay, I thought that consistency with the label was prefer because of the JASP syntax, but I don't think anyone is gonna use jaspSem with syntax anytime soon

Reply:
I mean if your suggestion is to change the label, I can do that